lib/health_check/middleware_health_check.rb

```diff
@@ -7,7 +7,8 @@ def initialize(app)
 
     def call(env)
       uri = env['PATH_INFO']
-      if uri.include? HealthCheck.uri
+      uriWithoutSuffix = uri[/([^\.]*)/,1]
```

Comment:
change to downcase pls sth like:  `uri_without_suffix`
also consider using gsub since it is more popular way for applying regex based operations
```
uri_without_suffix = '/abc/health2.xml'.gsub /\.[\w]{2,}$/, ''
```
removes 2 or longer suffixes